app/views/travel_advice/index.html.erb

```diff
@@ -27,16 +27,14 @@
             </div>
             <div class="travel-advice-notice__content">
               <p class="govuk-body">
-                Follow current COVID-19 rules where you live: <a href="/guidance/covid-19-coronavirus-restrictions-what-you-can-and-cannot-do" class="govuk-link">England</a>, <a href="https://www.gov.scot/coronavirus-covid-19/" class="govuk-link">Scotland</a>, <a href="https://gov.wales/coronavirus-travel" class="govuk-link">Wales</a> and <a href="https://www.nidirect.gov.uk/articles/coronavirus-covid-19-regulations-guidance-what-restrictions-mean-you" class="govuk-link">Northern Ireland</a>.
+                To understand the risks in a country, including the latest COVID restrictions (including for entry), follow FCDO Travel Advice.
               </p>
               <p class="govuk-body">
                 To prevent new COVID variants from entering the UK, you should not travel to <a href="/guidance/red-amber-and-green-list-rules-for-entering-england" class="govuk-link">amber or red list countries</a>.
+                From 19 July the <a href="/government/news/quarantine-free-travel-to-resume-on-19-july-for-fully-vaccinated-passengers-returning-from-amber-list-countries" class="govuk-link">guidance for amber countries will change</a>.
               </p>
               <p class="govuk-body">
-                To understand the risks in a country follow <a href="/guidance/travel-advice-novel-coronavirus" class="govuk-link">FCDO Travel Advice</a>.
-              </p>
-              <p class="govuk-body">
-                When you return, follow the rules to <a href="/uk-border-control" class="govuk-link">enter the UK from abroad</a> (except from Ireland).
+                Check what you need to do to <a href="//guidance/travel-abroad-from-england-during-coronavirus-covid-19" class="govuk-link">travel abroad and return to England</a>, or read travel guidance for <a href="https://www.gov.scot/publications/coronavirus-covid-19-guidance-on-travel-and-transport/" class="govuk-link">Scotland</a>, <a href="https://gov.wales/coronavirus-travel" class="govuk-link">Wales</a> or <a href="https://www.nidirect.gov.uk/articles/coronavirus-covid-19-travel-advice" class="govuk-link">Northern Ireland</a>.
```

Comment:
You've got a double slash in the first link on this line.

Reply:
Good spot! Updated and fixed up.